include/AutomationPattern.h

```diff
@@ -76,14 +76,17 @@ class EXPORT AutomationPattern : public TrackContentObject
 	MidiTime timeMapLength() const;
 	void updateLength();
 
-	MidiTime putValue( const MidiTime & _time, const float _value,
-						const bool _quant_pos = true );
+	MidiTime putValue( const MidiTime & time,
+				const float value,
+				const bool quantPos = true,
+				const bool controlKey = false );
```

Comment:
Core objects should not have references to the control key, which is a UI concept.